src/components/AnnotationBrowser/AnnotationList.vue

```diff
@@ -216,7 +263,16 @@ export default class AnnotationList extends Vue {
   }
 
   get filteredItems() {
-    return this.filterStore.filteredAnnotations.map(this.annotationToItem);
+    const query = this.filterQuery || ""; // Ensure that filterQuery is always treated as a string
+    return this.filterStore.filteredAnnotations
+      .filter(annotation => {
+        // If there is no query, return all annotations
+        if (!query.trim()) return true;
+
+        // Otherwise, filter by the id
+        return annotation.id.toString().includes(query.trim());
+      })
+      .map(this.annotationToItem);
```

Comment:
The issue with this approach is that you add a filter without adding it to the `filterStore`.
This will cause issues with other components that need to get the filtered annotations, for example when displaying only filtered annotations.
The right approach is to add a new filter "`idFilter`" in `filters.ts` and set it in `AnnotationFilters.vue`.
Do you need help for this?

Reply:
Ohhh, I see, yes, if you can do that, it would be great, thanks! I think it's a bit too complicated for me to figure out… I will probably make a lot of mistakes :)

Reply:
I guess I am wondering if it is better to keep it the way I implemented it. I think we would still want every other component that is accessing the item list to not worry about this particular filter, because this is meant to be just a quick "lookup" filter.

Reply:
Yes it is possible to have the "real" filters that work for all components and local filters only for the annotation list
But it should be clear for the users and in the code that it is a separate filter
If you think it is readable from a user point of view, we can leave the UI as it is
From a developer point of view, the filter is kind of "hardcoded" in the function, are you ok if I refactor it a bit?